mlir/test/Target/LLVMIR/llvmir.mlir

```diff
@@ -2621,3 +2621,62 @@ llvm.func @reqd_work_group_size() attributes {reqd_work_group_size = array<i32:
 llvm.func @intel_reqd_sub_group_size() attributes {intel_reqd_sub_group_size = 32 : i32}
 
 // CHECK: ![[#INTEL_REQD_SUB_GROUP_SIZE]] = !{i32 32}
+
+// -----
+
+llvm.func @foo()
+
+llvm.func @call_with_empty_opbundle() {
+  llvm.call @foo() [] : () -> ()
+  llvm.return
+}
+
+//      CHECK: define void @call_with_empty_opbundle() {
+// CHECK-NEXT:   call void @foo()
+// CHECK-NEXT:   ret void
+// CHECK-NEXT: }
```

Comment:
Could you also test:
```mlir
  llvm.call @foo() ["tag"()] : () -> ()
```

Reply:
Updated the parser and added the test.

Reply:
> Given the complexity of the printing and parsing it may also make sense to have some tests in the roundtrip.mlir to test the happy case?

This indeed caught some bugs in the printer for `llvm.invoke`! Updated.

Reply:
Yeah from experience printing and parsing is quite error prone...